selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -1401,4 +1401,9 @@ public void waitSuccessCondition(ExpectedCondition<Boolean> expression, int time
   public void waitSuccessCondition(ExpectedCondition<Boolean> expression) {
     waitSuccessCondition(expression, DEFAULT_TIMEOUT);
   }
+
+  /** Hides context menu. */
+  public void hideContextMenu() {
+    actionsFactory.createAction(seleniumWebDriver).moveByOffset(-1, -1).click().build().perform();
```

Comment:
What are we clicking here? Maybe <esc> would be better?

Reply:
To close context menu we have to click outside of it. So we move mouse not far from menu and perform click.

Reply:
You can simplify this:
seleniumWebDriverHelper.getAction().moveByOffset(-1, -1).click().perform();

Reply:
I was thinking of typing escape, but github took that as a tag :man_facepalming:

Reply:
Unfortunately escape does n't close context menu in che.